### ISSUE TITLE
External changes to experiment configuration

### ISSUE THREAD

**Issue by kklmn** [sardana-org/sardana#1226]
Hello,

I want to programmatically change ActiveMntGrp, do scans and return the previous measurement group. I can do it with setEnv. When I do it, a MessageBox pops up reporting "The experiment configuration has been modified externally".

Can I suppress it?

**Comment by reszelaz** [comment]
Similar question question was asked in https://github.com/sardana-org/sardana/issues/1207.
There is an "Auto update" mode of running expconf. This configuration is stored persistently so the next time you start your application the previous configuration will be respected. 

Sorry about the inconveniences!

**Comment by kklmn** [comment]
Thank you Zibi for giving the link!
I must add, it wasn't trivial to find this context menu. Only when you exactly know that it exists somewhere you keep right-clicking. Try it yourself.

**Comment by reszelaz** [comment]
I totally agree. We discussed it when this option was added: https://github.com/sardana-org/sardana/pull/1028#issuecomment-459667633. There I suggest improving the context menu organization.

But I would put efforts on implementing #1046 which IMO would remove 99% of the pop-up dialogs, maintaining the 1% of them where this pop-up could really save problems (we edit simultaneously to some external changes).
